build.sbt

```diff
@@ -202,7 +203,8 @@ lazy val explore: Project = project
         ReactGridLayout.value ++
         ReactHighcharts.value ++
         ReactResizable.value ++
-        ReactSizeMe.value
+        ReactSizeMe.value ++
+        Sttp.value
```

Comment:
This seems to be duplicated (line 202). Also, I'm not sure we need it in `explore` project if it is already in `common`.

Reply:
👍